flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,55 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
+	{
+		var arrayNames:Array<FlxAnimation> = [];
+		
+		for (i in _animations)
+		{
+			arrayNames.push(i);
+		}
+
+		return arrayNames;
+	}
+
+	public function nameList():Array<String>
```

Comment:
should be getNameList

Needs documentation

Reply:
Documentation given and changed the function, resolved, maybe